core/sdk-core/src/main/java/software/amazon/awssdk/core/client/handler/ClientExecutionParams.java

```diff
@@ -80,6 +82,15 @@ public InputT getInput() {
         return this;
     }
 
+    public RequestBody getRequestBody() {
+        return requestBody;
+    }
+
+    public ClientExecutionParams<InputT, OutputT> withRequestBody(RequestBody requestBody) {
```

Comment:
This is a protected API with with*/get*. Can we fix that really quick?

Reply:
Will update right before I merge.